functions/go/set-labels/transformer/labels.go

```diff
@@ -93,6 +93,11 @@ func (p *LabelTransformer) setLabelsInSpecs(o *fn.KubeObject) error {
 // Transform updates the labels in the right path using GVK filter and other configurable fields
 func (p *LabelTransformer) Transform(objects fn.KubeObjects) error {
 	for _, o := range objects {
+		// check if local config
+		localConfigPath := FieldPath{"metadata", "annotations", "config.kubernetes.io/local-config"}
```

Comment:
can we use [`IsLocalConfig`](https://pkg.go.dev/github.com/GoogleContainerTools/kpt-functions-sdk/go/fn#KubeObject.IsLocalConfig) and [`WhereNot`](https://pkg.go.dev/github.com/GoogleContainerTools/kpt-functions-sdk/go/fn#KubeObjects.WhereNot)

Reply:
`IsLocalConfig` is a method, and `WhereNot` asks for a function input. Do I need to create an extra function that converts method to function?